### ISSUE TITLE
Git Tag Needs Updating to v0.5.0

### ISSUE THREAD

**Issue by hydernaqvi** [ibmdb/go_ibm_db#250]
Most users don't have access to the latest changes and fixes.
Traditional commands such as the following will give the Go users version v0.4.5 which means most people are still using a version of this module from November 23, 2023. 

```
go mod tidy 
go install github.com/ibmdb/go_ibm_db/installer@latest
go install github.com/ibmdb/go_ibm_db/installer@v0.4.5
```

The above commands will not give the user the latest changes to the module.
For example the z/OS port won't be available.

[sys/unix](https://github.com/golang/sys) updates their  version tag about every month so users have the latest changes. We can go off of that example as it has similar use cases as this module.

Changing v0.4.5 to v0.5.0 would reflect a minor change to the module.

**Comment by hydernaqvi** [comment]
Until the tag is updated, users may have success using workspaces on z/OS. 
This blog goes into further detail: https://community.ibm.com/community/user/ibmz-and-linuxone/blogs/hyder-naqvi/2024/08/15/db2-for-go-for-zos?communityKey=6fb961a9-1e24-42c4-ad73-6a4a6b799f8a

**Comment by vmathur12** [comment]
@hydernaqvi
Released the code with new tag v0.5.0. Hence closing this issue.
Thanks
